### PR TITLE
PUB-1567 Add email for crime inactive user

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1567

### Change description ###

Send email to Crime user if he/she is inactive for x no of days.

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/UserNotificationService.java [hunk 1]
```diff
@@ -126,7 +126,9 @@ public String mediaUserRejectionEmailRequest(MediaRejectionEmail body) {
     public String inactiveUserNotificationEmailRequest(InactiveUserNotificationEmail body) {
         Templates emailTemplate = UserProvenances.PI_AAD.name().equals(body.getUserProvenance())
             ? Templates.INACTIVE_USER_NOTIFICATION_EMAIL_AAD
-            : Templates.INACTIVE_USER_NOTIFICATION_EMAIL_CFT;
+            : UserProvenances.CFT_IDAM.name().equals(body.getUserProvenance())
+            ? Templates.INACTIVE_USER_NOTIFICATION_EMAIL_CFT
```
Comment (KianKwa) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/UserNotificationService.java:130]: Nested ternary operators will cause sonarqube code smell. Better to split them up.

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/emailgeneration/useraccount/InactiveUserNotificationEmailGenerator.java [hunk 2]
```diff
@@ -25,7 +26,9 @@ public EmailToSend buildEmail(EmailData email, PersonalisationLinks personalisat
         InactiveUserNotificationEmailData emailData = (InactiveUserNotificationEmailData) email;
         Templates emailTemplate = UserProvenances.PI_AAD.name().equals(emailData.getUserProvenance())
             ? INACTIVE_USER_NOTIFICATION_EMAIL_AAD
-            : INACTIVE_USER_NOTIFICATION_EMAIL_CFT;
+            : UserProvenances.CFT_IDAM.name().equals(emailData.getUserProvenance())
+            ? INACTIVE_USER_NOTIFICATION_EMAIL_CFT
```
Comment (KianKwa) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/emailgeneration/useraccount/InactiveUserNotificationEmailGenerator.java:30]: Nested ternary operators will cause sonarqube code smell. Better to split them up.

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/notify/Templates.java [hunk 3]
```diff
@@ -86,6 +86,11 @@ public enum Templates {
                                          "Inactive IDAM account notification email",
                                          EmailLimit.STANDARD,
                                          new InactiveUserNotificationEmailGenerator()),
+
+    INACTIVE_USER_NOTIFICATION_EMAIL_CRIME("710a1ea4-226d-4e94-a8f7-5a102bb31612",
+                                         "Inactive IDAM account notification email",
```
Comment (KianKwa) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/notify/Templates.java:91]: Can we remain the description for this one and the CFT IDAM one differently to tell them apart?

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/UserNotificationService.java (hunk 1 above)
Comment (junaidiqbalmoj) [reply src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/UserNotificationService.java]: Thanks, created a new function and removed nested ternary operators (as suggested by Sonar) :) fixed.

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/emailgeneration/useraccount/InactiveUserNotificationEmailGenerator.java (hunk 2 above)
Comment (junaidiqbalmoj) [reply src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/emailgeneration/useraccount/InactiveUserNotificationEmailGenerator.java]: Thanks, created a new function and removed nested ternary operators (as suggested by Sonar) :) fixed.

**Review by KianKwa — APPROVED** [review]

**Pull request merged**